osu.Game/Users/UserActivity.cs

```diff
@@ -4,16 +4,24 @@
 #nullable disable
 
 using osu.Game.Beatmaps;
+using osu.Game.Configuration;
 using osu.Game.Graphics;
 using osu.Game.Online.Rooms;
 using osu.Game.Rulesets;
+using osu.Game.Scoring;
 using osuTK.Graphics;
 
 namespace osu.Game.Users
 {
     public abstract class UserActivity
     {
         public abstract string Status { get; }
+
+        /// <summary>
+        /// This property is used when the <see cref="DiscordRichPresenceMode"/> is <see cref="DiscordRichPresenceMode.Limited"/>
+        /// </summary>
+        public virtual string LimitedStatus => Status;
```

Comment:
This is feeling weird. A generic data class is referencing a third party component that isn't even valid outside of desktop? Something is inverted here.

I'd imagine there possibly being a `string GetStatus(bool hideIdentifiableInformation)` method or something instead.